utils.py

```diff
@@ -77,3 +79,159 @@ def decorated_function(*args, **kwargs):
         return decorated_function
 
     return decorator
+
+
+'''
+    Requirements for auth:
+        SwagLyrics PEM should be saved as "swag_pem" in environment variables
+        APP_ID is the id of the github app
+        repo_url is the api url of the repo this app is installed on
+'''
+
+APP_ID = os.getenv('swag_appid')
+
+# WARN: Security vulnerabilities - auth methods should eventually be moved
+# to its own class
+expiry_time, jwtoken = None, None
+
+repo_url = "https://api.github.com/repos/SwagLyrics/SwagLyrics-For-Spotify/"
+
+
+def produce_jwt() -> bytes:
```

Comment:
https://github.com/python/blurb_it/blob/054e564ba24f41f3223b79c7e0af1a1371d74881/blurb_it/util.py#L38 this might be a useful resource for this function